backend/functions/sms/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Flask==1.0.2
```

Comment:
Wondering why we need Flask. Is it a requirement for twilio?

Reply:
@dzx11 removed in latest PR. thanks for catching!